src/index.html

```diff
@@ -2,13 +2,69 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width,
+      user-scalable=no,
+      initial-scale=1.0,
+      maximum-scale=1.0,
+      minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <div class="logo">
+        <a href="#home"><img src="images/logo.png" alt="logo"></a>
+      </div>
+
+      <nav class="nav">
+
+        <ul class="nav__list">
+
+          <li class="link__item">
+            <a class="nav__link is-active" href="#">Apple</a>
```

Comment:
Add mock links to nav items
```suggestion
            <a class="nav__link is-active" href="#apple">Apple</a>
```